requirements-dev.txt

```diff
@@ -13,6 +13,6 @@ aioredis==1.3.1
 hiredis==1.1.0
 passlib==1.7.4
 cryptography==3.3.1
-aiohttp==3.7.3
+aiohttp<3.7
```

Comment:
I'll change this back (and the DeprecationWarning below) once 3.8 is released. This library is failing with 3.7 for the same reason as aiohttp-session.